tasks/idrac_update.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+    - name: Check if dsu is installed
```

Comment:
You do not need to check for that, https://github.com/CSCfi/ansible-role-dell-firmware-upgrade/blob/e5aa03dcfb56a849e8275b02f39840a864824d8c/tasks/main.yml#L61-L64 is making sure dsu exists